tests/unittests/Foundation/NSURLTests.m

```diff
@@ -225,16 +240,20 @@ void testNSURLMethod(SEL selector, NSURL* input, id argument, NSURL* expected) {
 
     // change current dir to app start up path
     ASSERT_TRUE(SetCurrentDirectoryW(startUpPath) != 0);
+    NSURL* baseURL = [NSURL fileURLWithPath:[manager currentDirectoryPath]];
+    NSURL* targetURL = [NSURL URLWithString:@"data/NSFileManagerUT.txt" relativeToURL:baseURL];
 #else
     char tempBuffer[_MAX_PATH];
     wcstombs(tempBuffer, startUpPath, _MAX_PATH);
 
     ASSERT_TRUE(chdir(dirname(tempBuffer)) == 0);
+    NSURL* baseURL = [NSURL fileURLWithPath:[manager currentDirectoryPath]];
+    NSURL* targetURL = [NSURL URLWithString:@"OSX/data/NSFileManagerUT.txt" relativeToURL:baseURL];
```

Comment:
Apparently my machine needs an update.  This could be a bug in the version I'm running, in which case I'll revert this.